app/deploy-cert-to-lb

```diff
@@ -64,11 +71,35 @@ if [ $? -eq 0 ]; then
     params+=(--ssl-verify-depth ${SSL_VERIFY_DEPTH})
     params+=(--ssl-verify-peer-certificate ${SSL_VERIFY_PEER_CERTIFICATE})
     params+=(--ssl-certificate-name ${CERT_NAME})
+    params+=(--cipher-suite-name ${CIPHER_SUITE_NAME});
+    params+=(--protocols ${PROTOCOLS});
+    params+=(--server-order-preference ${SERVER_ORDER_PREFERENCE});
     params+=(--force)
     params+=(--wait-for-state SUCCEEDED)
 
     oci lb listener update \
       ${params[@]}
+    
+    # Delete old certificates if listenet update succeeded
+    if [ $? -eq 0 ]; then
```

Comment:
Thanks for this. I deliberately left out any "destructive" steps in the scripts. Should we make this a configurable option that defaults to `false`?

Reply:
Yes, definitely. I will make change.

Reply:
Added variable DEL_OLD_CERTS to environment file example. Script checks that variable and deletes old certificates if value is Y and listener update did succeeded. 
Also noticed we might need define --auth instance_principal parameter for oci command calls if using dynamic groups. At least I didn't get it working without it

Reply:
Thanks for your quick turnaround and contributions. About the `--auth instance_principal` parameter, unless something had change, the environment variable `OCI_CLI_AUTH=instance_principal` set in the `Dockerfile` should have taken care of that.

Reply:
Yes, you are correct. I was just testing deploy-cert-to-lb script running it on my PC. Sorry about that.

Reply:
No worries. If you don't mind, I'll remove that in a future update.

Reply:
Sure. I already working on that https://github.com/jariolaine/oci-le-cert-manager/tree/patch-2